modules/openy_gc_log/src/LogArchiver.php

```diff
@@ -284,6 +284,9 @@ protected function prepareLogsForExport($logFileName = '') {
         $entity_type_id = $entity_type === 'node' ? 'node' : 'eventinstance';
         $entity = $this->entityTypeManager->getStorage($entity_type_id)
           ->load($entity_id);
+        if ($entity) {
```

Comment:
@froboy , @hamrant we need something like this in this place:
```suggestion
        if ($entity is not instanceof EntityInterface) {
```

Reply:
@hamrant I've updated it according to php standard
previous was producing fatal error:)